lib/assert.js

```diff
@@ -139,3 +139,21 @@ x.regex = function (contents, regex, msg) {
 x.ifError = x.error = function (err, msg) {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
+
+/*
+ * deprecated APIs
+ */
+x.same = function (val, expected, msg) {
+	deprecationNotice('same()', 'deepEqual()');
+	return x.deepEqual(val, expected, msg);
+};
+
+x.notSame = function (val, expected, msg) {
+	deprecationNotice('notSame()', 'notDeepEqual()');
+	return x.notDeepEqual(val, expected, msg);
+};
+
+function deprecationNotice(oldApi, newApi) {
+	console.warn(`DEPRECATION NOTICE: ${oldApi} has been renamed to ${newApi} and will eventually be removed`);
```

Comment:
Definitely. Would you like a more robust deprecation abstraction, or is this simple function and `console.warn` sufficient?